src/main/java/pages/createAccountPage/AccountFormUtils.java

```diff
@@ -0,0 +1,50 @@
+package pages.createAccountPage;
+
+import org.openqa.selenium.By;
+import pages.BasePage;
+
+public class AccountFormUtils {
+    protected BasePage basePage;
+
+    public AccountFormUtils() {
```

Comment:
I would add CreateAccountInput into the constructor here and declare all fields in page object - in this way you loose posibiity to easily access all the methods from AccountFormUtils() inside the tests:)

Reply:
I refactor some code using your advices:

- Change ```CreateAccountInput``` to ```CreateAccountFormId``` and ```AccountFormUtils``` to ```CreateAccountInput```
- Now for ```CreateAccountInput``` is responsible for handling a single input, where the id from ```CreateAccountFormId``` is provided in the constructor
- In ```CreateAccountPage``` I defined all inputs as fields by creating an object of the ```CreateAccountInput``` class
```java
    public static CreateAccountInput firstName = new CreateAccountInput(FIRST_NAME);
    public static CreateAccountInput lastName = new CreateAccountInput(LAST_NAME);
    public static CreateAccountInput email = new CreateAccountInput(EMAIL);
    public static CreateAccountInput telephone = new CreateAccountInput(TELEPHONE);
    public static CreateAccountInput fax = new CreateAccountInput(FAX);
    public static CreateAccountInput company = new CreateAccountInput(COMPANY);
    public static CreateAccountInput address1 = new CreateAccountInput(ADDRESS1);
    public static CreateAccountInput address2 = new CreateAccountInput(ADDRESS2);
    public static CreateAccountInput city = new CreateAccountInput(CITY);
    public static CreateAccountInput regionState = new CreateAccountInput(REGION_STATE);
    public static CreateAccountInput zipCode = new CreateAccountInput(ZIPCODE);
    public static CreateAccountInput country = new CreateAccountInput(COUNTRY);
    public static CreateAccountInput loginName = new CreateAccountInput(LOGIN_NAME);
    public static CreateAccountInput password = new CreateAccountInput(PASSWORD);
    public static CreateAccountInput confirmPassword = new CreateAccountInput(CONFIRM_PASSWORD);
    public static CreateAccountInput subscribeYes = new CreateAccountInput(SUBSCRIBE_YES);
    public static CreateAccountInput subscribeNo = new CreateAccountInput(SUBSCRIBE_NO);
    public static CreateAccountInput privacyPolicy = new CreateAccountInput(PRIVACY_POLICY);
```
- which it allows to simplify the ```fillCreateAccountForm``` method from ```CreateAccountPageSteps```
```java
    public static void fillCreateAccountForm(AccountModel accountModel) {
        createAccountPage = new CreateAccountPage();
        var customerModel = accountModel.customerModel;

        //Form: Your Personal Details
        firstName.fillInput(customerModel.firstName);
        lastName.fillInput(customerModel.lastName);
        email.fillInput(customerModel.email);
        telephone.fillInput(customerModel.phone);
        fax.fillInput(customerModel.fax);

        //Form: Your Address
        company.fillInput(customerModel.company);
        address1.fillInput(customerModel.address1);
        address2.fillInput(customerModel.address2);
        city.fillInput(customerModel.city);
        regionState.selectStateOrCountry(customerModel.state);
        zipCode.fillInput(customerModel.zipCode);

        //Form: Login Details
        loginName.fillInput(accountModel.loginName);
        password.fillInput(accountModel.password);
        confirmPassword.fillInput(accountModel.password);

        //Form: Newsletter
        createAccountPage.subscribeToNewsletter(accountModel.isSubscribedToNewsletter);
        createAccountPage.acceptPrivacyPolicy();
    }
```